cmd/common_test.go

```diff
@@ -410,11 +491,6 @@ func fakeExecutableInPath(execFile, execCode string) (restoreFn, error) {
 		return nil, err
 	}
 
-	err = os.Chdir(tmp)
```

Comment:
Curious why this was removed?

Reply:
It turns out I added the by mistake. We didn't have to navigate to the newly created `tmp` directory for the logic to work as the code refers to it using absolute paths. Removing it avoids errors with other Unit Tests that rely on being executed at the root of the source tree to find directories like `cfg`, etc...